Federiconardelli7/issue2068 remote content loading

federiconardelli7 opened the pull request:
remote-content.mts file updated to remote fetch files. 

this one: https://github.com/ava-labs/avalanchego/blob/master/api/keystore/
is not included because it's now deleted.

meaghanfitzgerald commented:
is the remote-content script removing the unused metadata from the files in avalanchego or did you manually changed the pulled content within this PR?
I.e. [This page](https://builders-hub-git-federiconardelli7-issue2068-re-a93c24-ava-labs.vercel.app/docs/nodes/chain-configs/p-chain) ideally needs the unused metadata (lines 1-6)  removed from the [file in avalanchego](https://github.com/ava-labs/avalanchego/blob/master/vms/platformvm/config/config.md?plain=1), however it looks like it was already removed.

federiconardelli7 commented:
The remote-content script removed the unused metadata (lines 1-6). I updated the MDX transformation to fetch the data correctly, and added missing entries for files that weren't being remote fetched before. For example, the page at https://build.avax.network/docs/nodes/configure/configs-flags is now fetched like this https://builders-hub-git-federiconardelli7-issue2068-re-a93c24-ava-labs.vercel.app/docs/nodes/configure/configs-flags

meaghanfitzgerald commented:
The page @federiconardelli7 mentioned has 2 title lines and 2 subtitles: https://build.avax.network/docs/nodes/configure/configs-flags
